packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -102,7 +102,7 @@ Example usage of the `nimble-rich-text-editor` in the application layer is as fo
 
 _Props/Attrs_
 
--   `markdownValue` - is an accessor used to get and set the markdown value.
+-   `markdown` - is an accessor used to get and set the markdown value.
     -   `getter` - this will serialize the content by extracting the Node from the editor and convert it into a markdown string using
         [prosemirror-markdown serializer](https://github.com/ProseMirror/prosemirror-markdown/blob/master/src/to_markdown.ts#L30).
     -   `setter` - this will parse the markdown string into a Node and load it back into the editor using
```

Comment:
Can you capture a discussion in an `Alternatives` section about the choice between getter and setter accessors vs `getMarkdown` and `setMarkdown` methods?

I can't remember how we decided to use accessors instead of methods so if you can capture the reasoning in the spec that would be helpful.

Reply:
@rajsite I have added the `Alternatives` section on how we choose `markdown` as an accessor instead of methods.

Reply:
Huh. That justification does not seem very convincing.. Just to be clear, setting the property on the viewer is NOT a one time operation right? The value can be set multiple times whenever the app needs to update the content. I think that statement is misleading and should be clarified.

The best trade-off I can think of is that the property syntax allows for one-way data binding which is easier for clients. However because calculating the markdown representation can be expensive an accessor was used instead of a property that is written to on changes for the editor component. The documentation will need to include discussion on how clients can use the component performantly and avoid reading the accessor unnecessarily.

Reply:
Apologies for the confusion caused. The one-time operation I meant was that the `viewer` component does not get the processed `markdown` value for every change like the `editor` component, instead, it returns the same markdown value that it receives from the client. However, it is misleading and not a convincing justification for this case. So I have removed the statement and updated it as you suggested.

Reply:
looks great! ty!